guides/common/modules/proc_attaching-satellite-infrastructure-subscription.adoc

```diff
@@ -3,7 +3,9 @@
 
 After you have registered {ProductName}, you must identify your subscription Pool ID and attach an available subscription.
 The {ProjectName} Infrastructure subscription provides access to the {ProjectName} and {RHEL} content.
+ifdef::satellite[]
 For {RHEL} 7, it also provides access to Red{nbsp}Hat Software Collections (RHSCL).
+endif::[]
```

Comment:
I think this is a bit redundant given this whole procedure is guarded by `ifdef::satellite[]` in the assemblies:
```console
$ rg proc_attaching-satellite-infrastructure -B 3
guides/common/assembly_installing-capsule-server.adoc
14-
15-ifdef::satellite[]
16-// Attaching the Satellite Infrastructure Subscription
17:include::modules/proc_attaching-satellite-infrastructure-subscription.adoc[leveloffset=+1]

guides/common/assembly_installing-server-connected.adoc
46-ifdef::satellite[]
47-include::modules/proc_registering-to-red-hat-subscription-management.adoc[leveloffset=+1]
48-
49:include::modules/proc_attaching-satellite-infrastructure-subscription.adoc[leveloffset=+1]
```

```suggestion
For {RHEL} 7, it also provides access to Red{nbsp}Hat Software Collections (RHSCL).
```